waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/model/AbstractMetaStore.java

```diff
@@ -152,6 +152,12 @@ public void setLatency(long latency) {
     this.latency = latency;
   }
 
+  public abstract List<String> getMappedDatabases();
```

Comment:
isn't the logic in the subclasses the same, can we move it here instead of abstract?

Reply:
If we move it there, then when the yaml file is overwritten, if the original yaml file had no line for mapped databases, then it will be overwritten to `mapped-databases: []`

Reply:
not if we allow null values, which I think we should